src/v3/src/components/Widget/index.tsx

```diff
@@ -304,9 +305,9 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note: we need to keep both themeproviders (MUI/ODS) until ODS exports all MUI components */}
       <MuiThemeProvider theme={brandedTheme}>
-        {/* the style is to allow the widget to inherit the parent's bg color */}
-        <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
-          <OdysseyCacheProvider>
+        <OdysseyCacheProvider nonce={cspNonce}>
+          {/* the style is to allow the widget to inherit the parent's bg color */}
+          <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
```

Comment:
This is still being imported directly from MUI. Once we upgrade odyssey to > `0.18` then we can pull the component from there.